assets/javascripts/kitten/karl/pages/homepage/partners/components/card.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { pxToRem, stepToRem } from '../../../../../helpers/utils/typography'
+import { Title } from '../../../../../components/typography/title'
+import { Text } from '../../../../../components/typography/text'
+import { ScreenConfig } from '../../../../../constants/screen-config'
+import { GridCol } from '../../../../../components/grid/grid'
+import { mediaQueries } from '../../../../../hoc/media-queries'
+import classNames from 'classnames'
+
+const StyledCard = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: ${pxToRem(20)};
+
+  @media (min-width: ${pxToRem(ScreenConfig.M.min)}) {
+    margin-bottom: ${pxToRem(30)};
+  }
+
+  @media (min-width: ${pxToRem(ScreenConfig.L.min)}) {
+    margin-bottom: ${pxToRem(40)};
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  .Partners__card__imageCol {
+    padding: 0;
+    padding-right: ${pxToRem(7.5)};
+
+    @media (min-width: ${pxToRem(ScreenConfig.M.min)}) {
+      padding-right: ${pxToRem(10)};
+    }
+
+    @media (min-width: ${pxToRem(ScreenConfig.L.min)}) {
+      padding-right: ${pxToRem(15)};
+    }
+  }
+
+  .Partners__card__image {
+    margin-bottom: ${pxToRem(20)};
+    margin-right: ${pxToRem(30)};
+    display: block;
+    width: 100%;
+
+    @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+      margin-bottom: ${pxToRem(30)};
+    }
+
+    @media (min-width: ${pxToRem(ScreenConfig.M.min)}) {
+      margin-bottom: ${pxToRem(0)};
+    }
+
+    :last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  .Partners__card__title {
+    font-size: ${stepToRem(-1)};
+
+    @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+      font-size: ${stepToRem(1)};
+      margin-bottom: ${pxToRem(10)};
+    }
+
+    @media (min-width: ${pxToRem(ScreenConfig.L.min)}) {
+      font-size: ${stepToRem(2)};
+    }
+  }
+
+  .Partners__card__text {
+    display: none;
+
+    @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+      font-size: ${stepToRem(-1)};
+      margin-top: ${pxToRem(10)};
+      margin-bottom: 0;
+      display: block;
+    }
+  }
+
+  .Partners__card__descriptionCol {
+    padding-right: 0;
+    padding-left: ${pxToRem(7.5)};
+
+    @media (min-width: ${pxToRem(ScreenConfig.M.min)}) {
```

Comment:
est-ce que c'est pas plutôt 
```suggestion
    @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
```